src/Repository/SkillRepository.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Adherent;
+use Doctrine\ORM\EntityRepository;
+
+class SkillRepository extends EntityRepository
+{
+    /**
+     * Finds all availebles skills for autocomplete that connected user has not yet chosen.
+     */
+    public function getSkillsForAutocomplete(string $term, Adherent $user)
+    {
+        $qbUserSkills = $this
+            ->createQueryBuilder('us')
+            ->select('us.name')
+            ->leftJoin('us.summary', 'cv')
+            ->andWhere('cv.member = :user')
```

Comment:
I think there will be an issue here, as skills could be typed differently. We should probably add a slug to the Skill entity and compare using that slug, to avoid having different cases creating different skills.